[2025-08-07T00:25:40.627+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-07T00:25:40.645+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_to_snowflake_loader.load_train_to_snowflake manual__2025-08-07T00:25:37.364800+00:00 [queued]>
[2025-08-07T00:25:40.651+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_to_snowflake_loader.load_train_to_snowflake manual__2025-08-07T00:25:37.364800+00:00 [queued]>
[2025-08-07T00:25:40.652+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-07T00:25:40.667+0000] {taskinstance.py:2330} INFO - Executing <Task(SnowflakeOperator): load_train_to_snowflake> on 2025-08-07 00:25:37.364800+00:00
[2025-08-07T00:25:40.686+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=8525) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-07T00:25:40.690+0000] {standard_task_runner.py:63} INFO - Started process 8544 to run task
[2025-08-07T00:25:40.692+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_to_snowflake_loader', 'load_train_to_snowflake', 'manual__2025-08-07T00:25:37.364800+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/load/s3_to_snowflake_loader.py', '--cfg-path', '/tmp/tmpx2yb0gr6']
[2025-08-07T00:25:40.699+0000] {standard_task_runner.py:91} INFO - Job 79: Subtask load_train_to_snowflake
[2025-08-07T00:25:40.724+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-07T00:25:40.780+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_to_snowflake_loader.load_train_to_snowflake manual__2025-08-07T00:25:37.364800+00:00 [running]> on host 3bffe12d583b
[2025-08-07T00:25:40.917+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_to_snowflake_loader' AIRFLOW_CTX_TASK_ID='load_train_to_snowflake' AIRFLOW_CTX_EXECUTION_DATE='2025-08-07T00:25:37.364800+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-07T00:25:37.364800+00:00'
[2025-08-07T00:25:40.923+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-07T00:25:40.959+0000] {sql.py:276} INFO - Executing: 
            COPY INTO RAW_TRAIN
            FROM 's3://favorita-retail-pipeline/raw/train.csv'
            CREDENTIALS = (
                AWS_KEY_ID= 'ASIAVX5O2PQE4CISITBL',
                AWS_SECRET_KEY= '***',
                AWS_TOKEN= '***'
            )
            FILE_FORMAT = (TYPE = 'CSV' FIELD_OPTIONALLY_ENCLOSED_BY = '"' SKIP_HEADER = 1);
            
[2025-08-07T00:25:40.978+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_id' for task execution.
[2025-08-07T00:25:41.011+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn_id' for task execution.
[2025-08-07T00:25:41.022+0000] {connection.py:399} INFO - Snowflake Connector for Python Version: 3.10.0, Python Version: 3.12.3, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-08-07T00:25:41.029+0000] {connection.py:1239} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-08-07T00:25:41.741+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-08-07T00:25:41.742+0000] {sql.py:470} INFO - Running statement: COPY INTO RAW_TRAIN
            FROM 's3://favorita-retail-pipeline/raw/train.csv'
            CREDENTIALS = (
                AWS_KEY_ID= 'ASIAVX5O2PQE4CISITBL',
                AWS_SECRET_KEY= '***',
                AWS_TOKEN= '***'
            )
            FILE_FORMAT = (TYPE = 'CSV' FIELD_OPTIONALLY_ENCLOSED_BY = '"' SKIP_HEADER = 1);, parameters: None
[2025-08-07T00:25:49.826+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-08-07T00:25:49.828+0000] {sql.py:479} INFO - Rows affected: 1
[2025-08-07T00:25:49.829+0000] {snowflake.py:407} INFO - Rows affected: 1
[2025-08-07T00:25:49.830+0000] {snowflake.py:408} INFO - Snowflake query id: 01be32f9-0000-29db-0000-003d698be0ad
[2025-08-07T00:25:50.026+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2025-08-07T00:25:50.027+0000] {connection.py:762} INFO - closed
[2025-08-07T00:25:50.123+0000] {connection.py:768} INFO - No async queries seem to be running, deleting session
[2025-08-07T00:25:50.216+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-07T00:25:50.271+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_to_snowflake_loader, task_id=load_train_to_snowflake, run_id=manual__2025-08-07T00:25:37.364800+00:00, execution_date=20250807T002537, start_date=20250807T002540, end_date=20250807T002550
[2025-08-07T00:25:50.331+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-07T00:25:50.349+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-07T00:25:50.350+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
